### PR TITLE
Website review

### PR DESCRIPTION
**Pull request by arl**
#### :question: What

Apply modifications after website review

#### :hammer: How to test

1. List all steps necessary;
2. To test this pull request.

#### :white_check_mark: Checklists

_This section contains a list of checklists for common uses, please delete the checklists that are useless for your current use case (or add another checklist if your use case isn't covered yet)._

- [ ] Is there unit/integration test coverage for all new and/or changed functionality added in this PR?
- [ ] Have the changes in this PR been functionally tested?
- [ ] Has `make gofmt-write` been run on the code?
- [ ] Has `make govet` been run on the code? Has the code been fixed accordingly to the output?
- [ ] Have the changes been added to the [CHANGELOG.md](./CHANGELOG.md) file?
- [ ] Have the steps in [CONTRIBUTING.md](./CONTRIBUTING.md) been followed to update a Go module?

### CODE REVIEW

File: content/en/docs/How Tos/baker_components.md
Comment (tommyblue) [inline content/en/docs/How Tos/baker_components.md:37]:
```suggestion
These fields list the components that are available to topologies. All components present
```

File: content/en/docs/How Tos/baker_components.md
Comment (tommyblue) [inline content/en/docs/How Tos/baker_components.md:100]:
```suggestion
The `User` field lists user-defined configurations structures which aren't strictly
```

File: content/en/docs/How Tos/baker_components.md
Comment (tommyblue) [inline content/en/docs/How Tos/baker_components.md:144]:
```suggestion
to all components so that they can use it at their will.
```

File: content/en/docs/How Tos/baker_components.md
Comment (tommyblue) [inline content/en/docs/How Tos/baker_components.md:165]: uhm, repetition

File: content/en/docs/How Tos/baker_components.md
Comment (tommyblue) [inline content/en/docs/How Tos/baker_components.md:189]:
```suggestion
with their names. This function exists to convert a field name to its index, it also controls
```

File: content/en/docs/How Tos/create_filter.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_filter.md:24]: I'd use `Filter` or **Filter**, but not *Filter* that, if I'm not wrong, we're not using anywhere else

File: content/en/docs/How Tos/create_input.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_input.md:42]: same as above, `Input` or **Input** IMO

File: content/en/docs/How Tos/create_output.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_output.md:18]: same

File: content/en/docs/How Tos/create_output.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_output.md:57]:
```suggestion
The `New` field in the `OutputDesc` object should be to assigned to a function that returns a new `Output`.
```

Comment (tommyblue) [reply content/en/docs/How Tos/create_output.md:57]: unless you want a newline, but I think it's a typo

File: content/en/docs/How Tos/create_upload.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_upload.md:19]: same

File: content/en/docs/How Tos/create_upload.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_upload.md:15]: same for *Upload*

File: content/en/docs/How Tos/create_upload.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_upload.md:51]:
```suggestion
The `New` field in the `UploadDesc` object should be to assigned to a function that returns a new `Upload`.
```

File: content/en/docs/How Tos/create_input.md
Comment (tommyblue) [inline content/en/docs/How Tos/create_input.md:78]:
```suggestion
The `New` field in the `InputDesc` object should be to assigned to a function that returns a new `Input`.
```

File: content/en/docs/How Tos/custom_record.md
Comment (tommyblue) [inline content/en/docs/How Tos/custom_record.md:16]: 2 times "Record interface" in 2 lines, I'd remove one

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:10]: `execution` is repeated twice, I'd probably remove `During its execution` at the beginning

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:13]:
```suggestion
files read or written, to performance statistics published by the Go 
```

Comment (tommyblue) [reply content/en/docs/How Tos/metrics.md:13]: I read it better without it

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:16]:
```suggestion
All components need to implement a `Stats` method where they can expose metrics. 
```

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:19]:
```suggestion
[`baker.MetricsBag`](https://pkg.go.dev/github.com/AdRoll/baker#MetricsBag),
```

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:38]: what do you think of adding a title for this section? Like "A practical example" or so

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:39]:
```suggestion
should be discarded, we might want to keep track of the requests' durations in an histogram.
```

File: content/en/docs/How Tos/metrics.md
Comment (tommyblue) [inline content/en/docs/How Tos/metrics.md:40]: can/would?

**Review by tommyblue — COMMENTED** [review]
added comments and suggestions, mostly for typos.
I'd remove the built files and will build only the `gh-pages` branch to avoid each PR bring their own built files, that aren't incompatible once merged unless each PR is the only one opened at the same time or each one is rebased before merging. A bit risky imo and building the `gh-pages` branch seems clearer to me (and we should automate it in the CI)

**Comment by arl** [comment]
@tommyblue I addressed your comments and pushed.
Don't merge yet as I still have to add some stuff

File: content/en/docs/How Tos/pipeline_configuration.md
Comment (tommyblue) [inline content/en/docs/How Tos/pipeline_configuration.md:14]: nit: I don't like `for instance` at the end

**Review by tommyblue — APPROVED** [review]

**Pull request merged**
